style.css

```diff
@@ -545,6 +556,38 @@ textarea:hover {
 
   #mobile-nav {
     display: flex;
+    z-index: 3;
+  }
+
+  .mobile-menu {
+    display: none;
+    position: fixed;
+    justify-content: left;
+    background: rgb(255, 255, 255);
+    height: 100vh;
+    width: 100%;
+    z-index: 2;
+    color: #3a4a42;
+    font-weight: 700;
+  }
```

Comment:
- [ ] Perfect :clap: you did a very nice job :+1: but there is some little design issues please stick to the Figma design your design is not the same as the Figma design check below :point_down: the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-04-11 04-00-10](https://user-images.githubusercontent.com/45226848/231016369-6c08dffa-2bc3-40f6-9327-105cd84cf7c8.png)

**Figma Design**
![Screenshot from 2023-04-11 03-59-55](https://user-images.githubusercontent.com/45226848/231016385-7f9dcced-25dd-47e3-b438-972f71380170.png)


- [ ] If you check your design the `cross` icon is very small so kindly increase the size of the `cross` icon :+1:
- [ ] If you check your `font style` is not the same as the Figma design kindly use the same font as mentioned in Figma Design :slightly_smiling_face:

Reply:
add all the changes. thank you for the good review